tests/test_mock2.py

```diff
@@ -0,0 +1,110 @@
+from connexion.utils import generate_example
+from re import fullmatch, Match
+
+
+def test_short_string():
```

Comment:
I think a better name would be `test_generate_example_string`. And similar below.